pkgs/applications/networking/browsers/microsoft-edge/update.py

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i python3 -p python3Packages.packaging python3Packages.debian
+
+import base64
+import gzip
+import textwrap
+from urllib import request
+
+from debian.deb822 import Packages
+from debian.debian_support import Version
+
+
+def packages():
+    packages_url = 'https://packages.microsoft.com/repos/edge/dists/stable/main/binary-amd64/Packages.gz'
```

Comment:
We could reduce some code if we use ``https://packages.microsoft.com/repos/edge/dists/stable/main/binary-amd64/Packages``